pycape/_attestation.py

```diff
@@ -32,7 +32,7 @@ def download_root_cert():
     return root_cert
 
 
-def parse_attestation(attestation, root_cert):
+def parse_attestation(attestation, root_cert=None):
```

Comment:
can we make this not the default? a user of this function might accidentally skip verifying the cert chain with the default call. if we're forcing ourselves to skip in the `cape.key` route we should make it harder for other callers to do that, not easier

Reply:
Works for me good point.